### DISCUSSION TITLE
[BUG] Auto close is required and the limit is too low

### DISCUSSION THREAD

**Discussion by GeckoBoy84** [discord-tickets/bot#463] [General]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

- [ ] Auto close is a required option
- [ ] Auto close has a maximum of 24 days

### Expected Behavior

- [ ] Should not be required
- [ ] Should allow it to be 30 days at minimum

### Environment

```markdown
- OS: Pterodactyl - Ubuntu Server 20.04
- Node: 18
- NPM: Unsure
- Bot: 4.0.6
```

**Comment by rakosi2** [comment]
Can we get a better title than literally "<title>" same with #452 and #457

**Comment by RooRay** [comment]
Converted to a discussion since this isn't really a bug and the issues that you are facing appear to be with your own installation, AFAIK.

**Comment by eartharoid** [comment]
@GeckoBoy84 The auto-close feature only applies to stale tickets, and by default, the only way for tickets to become stale is for someone to send a close request. This means the default behaviour is that tickets will only be closed if
1. someone requests it to be closed
2. the other party does not respond (reject it) within the time

I think 3 weeks is plenty of time to click a button.
